### DISCUSSION TITLE
Are nulls (eg double.NaN) stored, or missing as in firestore?

### DISCUSSION THREAD

**Discussion by lorriman** [isar/isar#404] [Help] [answered]
I have a specific sparse data need and with firestore the convention is simply that the field is missing in the underlying datastore if it were null. This suits me as it makes storage very efficient. 

So I'm wondering if Isar stores the double.NaN for a null, or like firestore it just misses out the field when storing/saving.

**Comment by simc** [comment, marked as answer]
@lorriman Isar stores `double.NaN` when a double field is null: https://isar.dev/schema.html#annotating-classes

**Reply by lorriman** [reply]
Thanks!
